src/main/resources/helptexts/formats.html

```diff
@@ -1,5 +1,17 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>Format</h1>
+<p>Software used.</p>
+<p>This field can be used to indicate which software and which file format was used to store the
+    data.
+    Indicate which software was used, and preferably also fill in the name of the manufacturer, the
+    product name, and the version </p>
+<p>Click the <i class="fas fa-plus-square"></i> button behind a field to add an additional field.
+</p>
+<h2>Examples</h2>
+<ul>
```

Comment:
These examples don't make sense. Are they in the document?

Reply:
changed to 
`Microsoft-Access, Office 2000
MapInfo 7.0
AutoDESK- AutoCAD 97
application/msword
image/jpeg
`